Tryouts/Plugins/Services/CommunicationsServices/GrpcCommunicationsService/SubscriptionClientImpl.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions;
+using NP.Utilities.Attributes;
+using Subscriptions;
+using static Subscriptions.SubscriptionsService;
+
+namespace MorganStanley.ComposeUI.Services.CommunicationsServices.GrpcCommunicationsService
+{
+    [Implements(typeof(ISubscriptionClient), isSingleton:true)]
+    public class SubscriptionClientImpl : ISubscriptionClient
+    {
+        private SubscriptionsServiceClient _client;
+        CancellationTokenSource _cts = new CancellationTokenSource();
+
+        public async Task Connect(string host, int port)
+        {
+            await Task.Delay(5000);
```

Comment:
Can you please explain why do you need this delay?

Reply:
I use this delay to let the server start. Of course, we have to come up with a better way of notifying the clients that the server is up, but for the tryouts I think it is ok. We won't move it to src until this problem is resolved.